barretenberg/cpp/src/barretenberg/translator_vm/translator_flavor.hpp

```diff
@@ -717,6 +720,13 @@ class TranslatorFlavor {
             result.insert(result.end(), special.begin(), special.end());
             return result;
         }
+        // Get witness polynomials including shifts. This getter is required by ZK-Sumcheck.
```

Comment:
so I see that across flavor classes these two functions have very different shapes, is there a way to make these more consistent?

Reply:
I guess, it would require re-shaping many things. The main problem is that in Mega or Ultra, there are shifted entities that are not witnesses (the tables), and I have to split shifted entities into witnesses and non-witnesses. Or derived entities are not a part of witness entities, although the evaluations of corresponding polynomials do leak private information